common/fw-update-helper.cpp

```diff
@@ -214,6 +214,12 @@ namespace rs2
         else
             serial = _dev.query_sensors().front().get_info(RS2_CAMERA_INFO_FIRMWARE_UPDATE_ID);
 
+        // TODO: DFU issue - remove d500_device usage when HKR FW team fix the firmware product id issue
+        // also, we set is_signed here to be true, because there is one flow for HKR, which is DFU (signed)
+        // if the file is unsigned or not compataible, HKR FW will notify for that.
+        bool d500_device = true;
```

Comment:
Let's please change this from hard coded to reading the product line and compare to "D500"

Reply:
We can place this 2 lines inside

Reply:
We can even change 
`bool d500_device = true;` to
`bool validate_fw_update_id = true;` and set as `false` if D500.
The motivation is to get minimal changes around the code that know D500, this way the future merge to development will be easier

Reply:
Fixed